examples/rust/ex1-volume/src/main.rs

```diff
@@ -0,0 +1,584 @@
+// Copyright (c) 2017-2018, Lawrence Livermore National Security, LLC.
```

Comment:
If we're using these copyright headers, we may as well get the year right on new code.

Note that they are not necessary from a licensing or IP standpoint and I'd have no problem skipping them entirely in source files.